src/main/java/com/google/servlets/NewAdviceServlet.java

```diff
@@ -1,23 +1,54 @@
 package com.google.servlets;
-
-import com.google.cloud.datastore.Datastore;
-import com.google.cloud.datastore.DatastoreOptions;
-import com.google.cloud.datastore.Entity;
-import com.google.cloud.datastore.FullEntity;
-import com.google.cloud.datastore.KeyFactory;
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.MultipartConfig;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import org.jsoup.Jsoup;
-import org.jsoup.safety.Whitelist;
 
-/** Servlet responsible for creating new advice */
-@WebServlet("/new-advice")
+/**
+ * Takes an image submitted by the user and uploads it to Cloud Storage, and then displays it as
+ * HTML in the response.
+ */
+@WebServlet("/advice")
+@MultipartConfig
 public class NewAdviceServlet extends HttpServlet {
-    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
-}
 
-// Modify your servlet file to take the data entered by users in your "contact me" 
-// form and store it as an entity in Datastore. Get this working before you worry about loading the data.
\ No newline at end of file
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+
+    // Get the form data entered by the user.
+    String username = request.getParameter("user-name");
+    String university = request.getParameter("university");
+    String major = request.getParameter("major");
+    String grade = request.getParameter("grade");
+    String advice = request.getParameter("advice");
```

Comment:
Note that in a real web application, you'd need to validate and/or cleanse any user entered text before doing anything with it. Some clever hacker might enter javascript or something!